pkg/daemon/server/service/rater/timestamped_counts.go

```diff
@@ -45,7 +46,14 @@ func (tc *TimestampedCounts) Update(podName string, count float64) {
 	tc.lock.Lock()
 	defer tc.lock.Unlock()
 	if count == CountNotAvailable {
-		delete(tc.podCounts, podName)
```

Comment:
won't this cause a leak? can't we garbage collect by adding a last-seen value? if not seen for the last x seconds (say 180), we can delete it.

Reply:
Good question. I don't think this will cause a leak. If a pod is turned down, the PodTracker will remove it from the activePod list. If a pod doesn't exist in the activePod list, then a task will never pick it up, meaning when we start the next round of populating TimestampedCounts, that pod won't be included.